libevm/VMSIMD.cpp

```diff
@@ -538,107 +537,106 @@ void VM::xget(uint8_t _b, uint8_t _c)
 	}
 }
 
-void VM::xput(uint8_t _b, uint8_t _c)
+void VM::xput(uint8_t _src_type, uint8_t _dst_type)
```

Comment:
Thought I'd search and destroyed all of these too...

Still - better than _b and _c.